packages/aws-cdk-lib/aws-appsync/lib/appsync-function.ts

```diff
@@ -45,6 +45,13 @@ export interface BaseAppsyncFunctionProps {
    * @default - no code is used
    */
   readonly code?: Code;
+  /**
```

Comment:
```suggestion
  readonly code?: Code;

  /**
```
Spacing. Can you please adjust for the other properties in this interface?